src/routes/(app)/admin[[hash=admin_hash]]/reporting/+page.svelte

```diff
@@ -442,7 +434,11 @@
 				</thead>
 				<tbody>
 					<tr class="hover:bg-gray-100 whitespace-nowrap">
-						<td class="border border-gray-300 px-4 py-2">{monthValue}/{yearValue}</td>
+						<td class="border border-gray-300 px-4 py-2"
+							>{beginsAtValue.toLocaleDateString('fr-FR')} -- {endsAtValue.toLocaleDateString(
+								'fr-FR'
+							)}</td
```

Comment:
Use `<time>` tag: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time

same in the rest of the file